peer.go

```diff
@@ -142,6 +154,10 @@ func newClientPeer(ctx context.Context, conn connection, clientRole uint64) (*Pe
 				k: roleParameterKey,
 				v: clientRole,
 			},
+			pathParameterKey: stringParameter{
```

Comment:
I think the parameter should only be set here if it is not empty. Otherwise, we would also set it for WebTransport sessions but with an empty value. I guess servers would have to close the connection in that case.

Reply:
This is now resolved in the latest commit